app/controllers/api/v1/construction_draws_controller.rb

```diff
@@ -0,0 +1,26 @@
+module Api::V1
+  class ConstructionDrawsController < ApplicationController
+    def index
+      render json: {
+        construction_draws: ConstructionDraw.paginate(page: page)
```

Comment:
this might be a style preference thing but i would load the information into an instance variable on the first line and then feed the paginated variable into the json render for readability.
ONLY if its easy to do